x/tunnel/types/encoding_packet_abi.go

```diff
@@ -0,0 +1,78 @@
+package types
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/abi"
+
+	feedstypes "github.com/bandprotocol/chain/v3/x/feeds/types"
+)
+
+var (
+	packetABI, _ = abi.NewType("tuple", "", []abi.ArgumentMarshaling{
```

Comment:
this is for Axelar only or for the others (in the future) as well? b/c i see you rename tss part (to have tss prefix), should it be align?

Reply:
It common use. not only axelar